static/app/components/teamSelector.tsx

```diff
@@ -46,6 +48,20 @@ const unassignedOption = {
   disabled: false,
 };
 
+const CREATE_TEAM_VALUE = 'CREATE_TEAM_VALUE';
+const createOption = {
+  value: CREATE_TEAM_VALUE,
+  label: 'Create team',
+  leadingItems: <IconAdd isCircled />,
+  searchKey: 'create',
+  actor: null,
+  disabled: false,
+};
```

Comment:
isn't it possible to have an extra option for creation without needing to pass this object?

Reply:
react-select has a create functionality, but only from the input value. (e.g. `Create 'test-org'`)
I tried it at first, but was not able to get it too work with an empty input.